src/TicTacToeLoop.py

```diff
@@ -0,0 +1,48 @@
+import sys 
+import numpy as np
+
+from ComputerTurn import getComputerInput
+from UserTurn import getUserMove
+from WinningCombo import checkForWin
+
+#  Game runner Responsibilities
+def checkForCatsGame(inputArray):
+    if len(inputArray) == 0:
+        return True
+
+    else:
+        return False
+
+def printBoard(gameBoard):
+    print("\n %s | %s | %s \n---+---+---\n %s | %s | %s \n---+---+---\n %s | %s | %s \n" % \
+    (gameBoard[0][0], gameBoard[0][1], gameBoard[0][2],
+    gameBoard[1][0], gameBoard[1][1], gameBoard[1][2],
+    gameBoard[2][0], gameBoard[2][1], gameBoard[2][2]))
+
+def oneTurn(gameBoard, inputArray):
+    printBoard(gameBoard)
+    
+    gameBoard, inputArray = getUserMove(gameBoard, inputArray)
+
+    if checkForCatsGame(inputArray) == True:
+        return gameBoard, inputArray
+
+    if checkForWin(gameBoard) == True:
+        return gameBoard, inputArray
+
+    gameBoard, inputArray = getComputerInput(gameBoard)
+
+    return gameBoard, inputArray
+
+def takeTurns(board, possibleInputs):
```

Comment:
Another small-ish thing. I _think_ that `takeTurns` is the only function in this file that's used by clients. Would be nice to see it at the top of the file. In the future (in a language like Java), you may want to actually make some of these other functions private. This will make them hard to test (and may force you to put them in separate files) but the benefit is that you can change the code  later on without worrying about breaking someone's code that relies on them.

Reply:
Do you mean by adding a _ before the function name like Javascript getters? I think python doesn't have private functions but I can keep that in mind as we progress!